### PR TITLE
Fix/210/history map add create and delete logic to streetcode coordinate

### PR DESCRIPTION
**Pull request by HlibBondarev**
History map add create and delete logic to streetcode coordinate

### CODE REVIEW

File: Streetcode/Streetcode.BLL/Mapping/AdditionalContent/Coordinates/StreetcodeCoordinateProfile.cs [hunk 1]
```diff
@@ -9,5 +10,7 @@ public class StreetcodeCoordinateProfile : Profile
    public StreetcodeCoordinateProfile()
    {
         CreateMap<StreetcodeCoordinate, StreetcodeCoordinateDto>().ReverseMap();
-   }
+        CreateMap<CreateStreetcodeCoordinateRequestDto, StreetcodeCoordinate>().ReverseMap();
+        CreateMap<StreetcodeCoordinate, CreateStreetcodeCoordinateResponseDto>().ReverseMap();
+    }
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/Mapping/AdditionalContent/Coordinates/StreetcodeCoordinateProfile.cs:15]: ReverseMap is not needed in both cases

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/StreetcodeCoordinate/Create/CreateStreetcodeCoordinateRequestDtoValidator.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        RuleFor(dto => dto.StreetcodeId)
+            .GreaterThan(0);
+
+        RuleFor(dto => dto.Latitude)
+            .GreaterThan(0);
+
+        RuleFor(dto => dto.Longtitude)
+            .GreaterThan(0);
+    }
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/MediatR/AdditionalContent/StreetcodeCoordinate/Create/CreateStreetcodeCoordinateRequestDtoValidator.cs:18]: negative longitude and latitude are valid. the thing is this point is definitely not in Ukraine

**Review by yaninavoda — APPROVED** [review]
I left a couple of comments

**Review by NMarteniuk — APPROVED** [review]

File: Streetcode/Streetcode.BLL/Mapping/AdditionalContent/Coordinates/StreetcodeCoordinateProfile.cs (hunk 1 above)
Comment (HlibBondarev) [reply Streetcode/Streetcode.BLL/Mapping/AdditionalContent/Coordinates/StreetcodeCoordinateProfile.cs:15]: everything is fixed -

Screenshot_89

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/StreetcodeCoordinate/Create/CreateStreetcodeCoordinateRequestDtoValidator.cs (hunk 2 above)
Comment (HlibBondarev) [reply Streetcode/Streetcode.BLL/MediatR/AdditionalContent/StreetcodeCoordinate/Create/CreateStreetcodeCoordinateRequestDtoValidator.cs:20]: everything is fixed

Screenshot_90

**Comment by HlibBondarev** [comment]
> I left a couple of comments

I also redid the validation tests

**Pull request merged**